### PR TITLE
Allow for symfony/console v2 or v3

### LINKED ISSUE #30: Update symfony/console
Please update symfony console dependency:
"symfony/console": "^2.1|^3.0",

### PR DESCRIPTION
**Pull request by keradus**
Closes #30

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,10 +5,11 @@
     "homepage": "https://github.com/clue/graph-composer",
     "license": "MIT",
     "require": {
-        "clue/graph": "~0.9.0",
+        "php": "^5.3.6 || ^7.0",
```
Comment (keradus) [inline composer.json]: guessed by travis... 
this one is required version for installing the package as dependency of other package

File: composer.json [hunk 2]
```diff
@@ -18,5 +19,10 @@
         "phar": {
             "bundler": "composer"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "5.3.6"
```
Comment (keradus) [inline composer.json]: and this one is used for lock file (so you won't lock to version of dependency that works on 5.6 but not 5.4)

**Comment by clue** [comment]
Nice one, thanks for the elaborate description! :+1:

**Comment by keradus** [comment]
great, thanks for merge !

**Pull request merged**